group_project_v2/project_api/dtos.py

```diff
@@ -25,7 +25,7 @@ class UserDetails(ReducedUserDetails):
     def __init__(self, **kwargs):
         super(UserDetails, self).__init__(**kwargs)
         self.gender = kwargs.get('gender', None)
-        self.avatar_url = kwargs.get('avatar_url', None)
+        self.profile_image_url = kwargs.get('profile_image', {}).get('image_url_small', None)
```

Comment:
@msaqib52 In my testing, this looks quite blurry, because `image_url_small` is a 30px image, stretched to 40px here. Can we change this to `image_url_medium` or `image_url_large` ?

Reply:
Comparison (right is `medium`):
![screen shot 2017-10-19 at 2 01 09 pm](https://user-images.githubusercontent.com/945577/31794098-febaef60-b4d5-11e7-85b6-31a2ed46a639.png)